apps/api/src/app/activity/activity.controller.ts

```diff
@@ -21,48 +24,56 @@ export class ActivityController {
   ) {}
 
   @Get('')
+  @ApiOkResponse({
+    type: ActivitiesResponseDto,
+  })
+  @ApiOperation({
+    summary: 'Get activity feed',
+  })
   @UseGuards(JwtAuthGuard)
   getActivityFeed(
     @UserSession() user: IJwtPayload,
-    @Query('channels') channels: ChannelTypeEnum[] | ChannelTypeEnum,
-    @Query('templates') templates: string[] | string,
-    @Query('emails') emails: string | string[],
-    @Query('search') search: string,
-    @Query('page') page = 0
-  ) {
+    @Query() query: ActivitesRequestDto
```

Comment:
Does swagger treat it properly as Query Params? And also are we handling it correctly on the docs site?

Reply:
Swagger treats this properly and it looks like is shown as they where spread as before...